### PR TITLE
Jessie new post

### PR DESCRIPTION
**Pull request by JessieWang0706**
## Overview
New post screen
Add Image from gallary
Delete Image
Details Screen
price number pad

## Test Coverage
Android pixel 32, api29

## Next Steps (delete if not applicable)
Networking stuff

## Screenshots (delete if not applicable)

https://user-images.githubusercontent.com/28740646/164817857-74681b49-0866-4bff-bc8a-869beb128cf4.MP4

### CODE REVIEW

**Review by Npikielny — COMMENTED** [review]
I don't think I'm supposed to be here

File: components/NegotiationModal.tsx [hunk 1]
```diff
@@ -10,31 +10,63 @@ export function NegotiationModal({
...
+              width: "100%",
+              marginBottom: 20,
+              elevation: 5,
+              alignItems: "center",
+            }}
+          >
+            <NegotiationProductBubble
+              product={"Blue Pants"}
+              price={"14.00"}
+              image={require("../assets/images/Pants.png")}
```
Comment (Haichen-Wang) [inline components/NegotiationModal.tsx:42]: don't forget to adjust this in networking

File: data/filter.tsx [hunk 2]
```diff
@@ -12,3 +12,13 @@ export const FILTER = [
...
 ];
+export const FILTER1 = [
+  {
+    id: 1,
+    title: "Funiture",
+  },
+  { id: 2, title: "Books" },
+  { id: 3, title: "Electronics" },
+  { id: 4, title: "Clothes" },
+  { id: 5, title: "Musical Instruments" },
```
Comment (Haichen-Wang) [inline data/filter.tsx:23]: this is inconsistent with categories we have on the backend. is this just for testing?

File: screens/NewPostImage.tsx [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+  const [image, setImage] = useState([]);
+  const [activeSlide, setActiveSlide] = useState(0);
+  const [refresh, setFresh] = useState(false);
+  const _carousel = useRef(null);
+
+  const pickImage = async () => {
+    let result = await ImagePicker.launchImageLibraryAsync({
+      mediaTypes: ImagePicker.MediaTypeOptions.All,
+      allowsEditing: true,
+      aspect: [4, 3],
```
Comment (Haichen-Wang) [inline screens/NewPostImage.tsx:39]: do we only allow people to crop their images to 4:3?

File: screens/NewPostImage.tsx [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+            if (image.length == 2) {
+              setImage([]);
+            } else {
+              image.splice(index, 1);
+              setImage(image);
+              setFresh(!refresh);
+            }
+          }}
+        >
+          <Feather name="trash" size={28} color="white" />
```
Comment (Haichen-Wang) [inline screens/NewPostImage.tsx:89]: trash doesn't work rn right? if not we should comment this out otherwise people will think it's a bug

**Review by Haichen-Wang — APPROVED** [review]
Wow, this is amazing!! You paid attention to details, I'm especially impressed by how the price field gets highlighted on entering the price! I've made some small comments.

**Pull request merged**